spec/features/workflow/schedule_spec.rb

```diff
@@ -10,6 +10,16 @@
     when_i_choose_a_scheduled_date_and_time
     and_i_click_save
     then_i_see_the_edition_has_a_set_scheduled_publishing_date_and_time
+
+    when_i_click_on_schedule
+    and_i_select_the_content_has_been_reviewed_option
+    and_click_publish
+    then_i_see_a_confirmation_that_the_edition_has_been_scheduled
+
+    when_i_visit_the_summary_page
+    then_i_see_the_edition_has_been_scheduled
+    and_i_can_no_longer_see_a_schedule_action
+    and_i_can_no_longer_edit_the_content
```

Comment:
This feature test seems to have got rather long. I wonder if it's actually testing two distinct features: setting a scheduled_publishing_datetime and scheduling an edition. What do you think?

Reply:
Now split into two test files.